src/main/java/com/flightstats/hub/metrics/StatsDFilter.java

```diff
@@ -67,8 +69,10 @@ public boolean isTestChannel(String channel) {
         return channel.toLowerCase().startsWith("test_");
     }
 
-    public Set<String> getRequestMetricsToIgnore() {
-        return requestMetricsToIgnore;
+    public boolean isIgnoredRequestMetric(RequestMetric metric) {
+        return metric.getMetricName()
+                .map(requestMetricsToIgnore::contains)
+                .orElse(true);
```

Comment:
I think you mean `false` here?

Reply:
Nevermind, this is saying "if there's no metric name, ignore it"

Reply:
I had `false` here initially but one of my stupid tests failed in protest. 😂